### PR TITLE
Added Bluemagic spells and missing regular spells

### PR DESCRIPTION
**Pull request by Dirklionheart**
Added: Pinecone Bomb
Added: Voracious Trunk

### CODE REVIEW

File: scripts/globals/spells/absorb-attri.lua [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+		spell:setMsg(85); -- Resists
+	else
+		if(caster:hasStatusEffect(EFFECT_NETHER_VOID)) then
+			caster:delStatusEffect(EFFECT_NETHER_VOID);
+			netherVoid = 1;
+		end
+		local effect1 = target:copyStatusEffect();
+		local effect = target:dispelStatusEffect(EFFECTFLAG_DISPELABLE);
+		if(effect1 == nil) then
+			printf("Null \n");
```
Comment (teschnei) [inline scripts/globals/spells/absorb-attri.lua]: can get rid of that print :)

File: scripts/globals/spells/animus_augeo.lua [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+function onSpellCast(caster,target,spell)
+
+   	local power = 20;
+	local duration = 180;
+
+	-- Estoqueurs Bonus
+	duration = duration + (duration * caster:getMod(MOD_ENHANCING_DUR));
+	
+	target:delStatusEffect(EFFECT_ENMITY_BOOST);
```
Comment (teschnei) [inline scripts/globals/spells/animus_augeo.lua]: also should delete Pax

File: scripts/globals/spells/animus_minuo.lua [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+function onSpellCast(caster,target,spell)
+
+   	local power = -10;
+	local duration = 180;
+
+	-- Estoqueurs Bonus
+	duration = duration + (duration * caster:getMod(MOD_ENHANCING_DUR));
+	
+	target:delStatusEffect(EFFECT_PAX);
```
Comment (teschnei) [inline scripts/globals/spells/animus_minuo.lua]: similarly to augeo, should also remove emnity boost

File: scripts/globals/spells/blizzaga_iv.lua [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+-- OnSpellCast
+-----------------------------------------
+
+function OnMagicCastingCheck(caster,target,spell)
+	return 0;
+end;
+
+function onSpellCast(caster,target,spell)
+	--doElementalNuke(V,M,caster,spell,target,hasMultipleTargetReduction,resistBonus)
+	local dmg = doElementalNuke(934,2,caster,spell,target,false,1.0);
```
Comment (teschnei) [inline scripts/globals/spells/blizzaga_iv.lua]: pretty sure -ga nukes should have hasMultipleTargetReduction = true, no?

**Comment by teschnei** [comment]
i'd like to see the items that are applying mods to pet removed - i've got plans to make that a lot better

**Comment by sisternERA** [comment]
i do see that the O is capitalized in the OnUseAbility.  i think current version requires it to be not capitalized

**Comment by xdemolish** [comment]
can't merge (and changes haven't been made in like 3 weeks)

**Pull request closed without merging**